charts/snapshot-controller/templates/_helpers.tpl

```diff
@@ -49,9 +49,78 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 Create the name of the service account to use
 */}}
 {{- define "snapshot-controller.serviceAccountName" -}}
-{{- if .Values.serviceAccount.create -}}
-    {{ default (include "snapshot-controller.fullname" .) .Values.serviceAccount.name }}
+{{- if .Values.controller.serviceAccount.create -}}
+    {{ default (include "snapshot-controller.fullname" .) .Values.controller.serviceAccount.name }}
 {{- else -}}
-    {{ default "default" .Values.serviceAccount.name }}
+    {{ default "default" .Values.controller.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "snapshot-validation-webhook.name" -}}
+{{- "snapshot-validation-webhook" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "snapshot-validation-webhook.fullname" -}}
+{{- if contains .Chart.Name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
```

Comment:
When I try to install the chart, I get 
```
Error: INSTALLATION FAILED: serviceaccounts "snapshot-controller" already exists
```

I believe this default needs to be changed to include a `-webhook` suffix or similar.

Reply:
@WanzenBug If the deployment name is "snapshot-controller" then everything becomes "snapshot-controller" without any suffix/prefix for the *snapshot-validation-webhook* components.
Will pin it to `snapshot-validation-webhook` in that case.